src/Fields/FieldWrapper.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactNode } from 'react';
+import {
+  useController,
+  FieldValues,
+  UseControllerProps,
+  FieldPathValue,
+  FieldPath,
+  UnpackNestedValue,
+} from 'react-hook-form';
+
+import { Form, FormItemProps } from '../Form';
+
+export type FieldWrapperProps<
+  TFieldValues extends FieldValues,
+  TName extends FieldPath<TFieldValues>,
+> = {
+  children: ReactNode;
+  controller: UseControllerProps<TFieldValues, TName>;
+  formItem?: FormItemProps<
+    UnpackNestedValue<FieldPathValue<TFieldValues, TName>>
+  >;
+};
+
+export function FieldWrapper<
+  TFieldValues extends FieldValues,
+  TName extends FieldPath<TFieldValues>,
+>(props: FieldWrapperProps<TFieldValues, TName>) {
+  const { fieldState } = useController<TFieldValues, TName>(props.controller);
+
+  return (
+    <Form.Item
+      // hasFeedback={fieldState.invalid} TODO uncomment when antd3 has been removed from limes-frontend
```

Comment:
can we solve this somehow via the FormProvider? In NeMo we have no dependency on antd3.

Reply:
Yes good idea to make it configurable with the `FieldProvider`.  Context could be extended to:


```
{
  disabled: boolean;
  formItemProps:  (fieldState: ControllerFieldState) => Partial<FormItemProps>;
}
```

Reply:
@mic-web you can just push to this pull request.

Reply:
Alternative to `Partial` might be possible

Reply:
Done